src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Animations/AnimationBehavior.shared.cs

```diff
@@ -37,14 +37,15 @@ protected override void OnDetachingFrom(VisualElement bindable)
 			base.OnDetachingFrom(bindable);
 		}
 
-		protected override async void OnTriggerHandled(object sender = null, object eventArgs = null)
+		protected override async void OnTriggerHandled(object? sender = null, object? eventArgs = null)
 		{
 			if (isAnimating)
 				return;
 
 			isAnimating = true;
 
-			await AnimationType?.Animate((View)sender);
```

Comment:
Both `AnimationType` and `Command` are not null there

```cs
await AnimationType.Animate((View?)sender);

if (Command.CanExecute(CommandParameter))
    Command.Execute(CommandParameter);
```

Reply:
Ah good catch - they _should_ be nullable.

Properties that use BindableObject should be nullable because the `deaultValue` of `BindableProperty.Create` is `null`.

<img width="1004" alt="Screen Shot 2021-03-05 at 3 13 45 PM" src="https://user-images.githubusercontent.com/13558917/110184846-b1cd8e80-7dc5-11eb-9a4b-f3492c62a98b.png">